src/index.html

```diff
@@ -1,12 +1,330 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div>
+        <a href="./" class="header-logo">
+          <img
+            class="header-logo-img"
+            src="images/logo.png"
+            alt="LOGO">
+        </a>
+
+      </div>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">samsung</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              class="link"
+              data-qa="nav-hover">
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">гаджеты</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">планшеты</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">фото</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="items-cards">
+        <div class="cards-container items-cards__cards-container">
+          <div class="product-card" data-qa="card">
+            <img
+              class="product-card__image"
+              src="images/imac.jpeg"
+              alt="imac photo">
+
+            <h1 class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+
+            <div class="product-card__code">Код товара: 195434</div>
+
+            <div class="product-card__reviews">
+              <div class="stars">
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+              </div>
+              <div class="feedbacks">Отзывов: 5</div>
+            </div>
+
+            <div class="price">
+              <div class="price__title">Цена:</div>
+              <div class="price__figure">69 999 грн</div>
+            </div>
+
+            <a href="https://ukr.net"
```

Comment:
```suggestion
            <a href="#"
```
Думаю решетки хватит

Reply:
ок